rocAL_pybind/rocal_pybind.cpp

```diff
@@ -851,7 +867,12 @@ py::class_<rocalListOfTensorList>(m, "rocalListOfTensorList")
     // rocal_api_data_transfer.h
     m.def("rocalToTensor", &wrapper_copy_to_tensor);
     m.def("getOutputTensors", [](RocalContext context) {
-        rocalTensorList *output_tensor_list = rocalGetOutputTensors(context);
+        rocalTensorList *output_tensor_list = nullptr;
+        {
+            // May block on ring buffer - release GIL while waiting
+            py::gil_scoped_release release;
```

Comment:
This statement is added in python end for most pipeline related functions, it is better to run training with this branch to ensure the perf is not affected in any way @SundarRajan28